src/app/campaign/status/campaign-status.component.ts

```diff
@@ -22,11 +22,11 @@ import { FlightState } from '../store/models';
       <grove-campaign-report [campaignName]="campaignName" [flights]="flights" [reportData]="reportData"></grove-campaign-report>
       <button mat-menu-item [disabled]="!canDuplicate" (click)="onDuplicate()">
         <mat-icon aria-hidden>file_copy</mat-icon>
-        <span>Duplicate</span>
+        <span>Duplicate Campaign</span>
```

Comment:
+10 i think this was a source of confusion.

Reply:
Added it to match the wording of the Delete Flight button. Thought it was best to be clear about what is being deleted.